digital-twin/device/src/test/java/com/microsoft/azure/sdk/iot/digitaltwin/device/AbstractDigitalTwinComponentTest.java

```diff
@@ -41,8 +41,7 @@
 public class AbstractDigitalTwinComponentTest {
     private static final String DIGITAL_TWIN_COMPONENT_NAME = "DIGITAL_TWIN_COMPONENT_NAME";
     private static final String DIGITAL_TWIN_INTERFACE_ID = "DIGITAL_TWIN_INTERFACE_ID";
-    private static final String TELEMETRY_NAME = "TELEMETRY_NAME";
-    private static final String TELEMETRY_PAYLOAD = "TELEMETRY_PAYLOAD";
+    private static final String TELEMETRY_PAYLOAD = "{\"TELEMETRY_NAME\": \"TELEMETRY_PAYLOAD\"}";
```

Comment:
The value of the telemetry is the same name as the entire thing, which is confusing. Suggest changing this to TELEMETRY_VALUE.

Reply:
fixed